test/com/google/enterprise/adaptor/AccumulatingDocIdPusher.java

```diff
@@ -60,8 +60,8 @@ public DocId pushNamedResources(Map<DocId, Acl> resources,
   @Override
   public GroupPrincipal pushGroupDefinitions(
       Map<GroupPrincipal, ? extends Collection<Principal>> defs,
-      boolean caseSensitive, ExceptionHandler handler)
-      throws InterruptedException {
+      boolean caseSensitive, boolean incremental, String sourceName,
+      ExceptionHandler exceptionHandler) throws InterruptedException {
```

Comment:
PJ seems to be resistant to sharing test utility classes. But at least (unlike util classes) there is a somewhat standard place to put them: testing.